samples/graph-change-notification-team-channel/csharp/README.md

```diff
@@ -0,0 +1,105 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates the use of team/Channel subscription that will post notifications when user create/edit/delete team/channel.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions 
+ contentType: samples
+ createdDate: "25-08-2022 04:30:15"
+urlFragment: officedev-microsoft-teams-samples-graph-change-notification-team-channel
+---
+
+# This is a sample application that shows the usage of change notifications for team and channel using Microsoft Graph.
+
+This is a sample application which demonstrates use of Team/Channel subscription that will post notifications when user create/edit/delete team/channel.
+
+## Concepts introduced in this sample
+- After successfully Setup of tab in teams you will get a welcome message and the subscription will be created for team/channel.
```

Comment:
Where are we going to get the welcome message ?

Reply:
when channel/team selected first this will come please check it below image
<img width="668" alt="3" src="https://user-images.githubusercontent.com/109147706/187635372-b1eba28c-d6f1-4d85-b7fa-71e24633b731.PNG">